calico-vpp-agent/policy/policy_server.go

```diff
@@ -1677,50 +1674,49 @@ func (s *Server) createAllowFromHostPolicy() (err error) {
 }
 
 func (s *Server) createEndpointToHostPolicy( /*may be return*/ ) (err error) {
-	pol := &Policy{
+	workloadsToHostPolicy := &Policy{
 		Policy: &types.Policy{},
 		VppID:  types.InvalidID,
 	}
-	r_deny_workloads := &Rule{
+	action := types.ActionDeny
+	if s.getEndpointToHostAction() == "ACCEPT" {
```

Comment:
`s.getEndpointToHostAction()` could directly return `types.ActionAllow` or `types.ActionDeny` that would allow to simplify part of the logic